OpenRA.Mods.Common/Scripting/Properties/MobileProperties.cs

```diff
@@ -36,7 +36,7 @@ public void Move(CPos cell, int closeEnough = 0)
 		}
 
 		[ScriptActorPropertyActivity]
-		[Desc("Moves within the cell grid, ignoring lane biases.")]
```

Comment:
The sole purpose of this Lua API (as opposed to the standard Move just above) is to turn off the lane bias - we need this to be able to script groups of units moving together side-by-side.

Reply:
I don't think that is very important (`ScriptedMove` is only used for water transports in TD and in Exodus and Infiltration in RA), but admit I haven't specifically tested this (the first GDI mission seemed fine though).

Reply:
Would rather play it safe, so have restored the lane bias option.